mega-zep-frontend/src/main/angular/frontend/src/app/modules/monthly-report/components/general-info/genInfoData.ts

```diff
@@ -0,0 +1,9 @@
+export class genInfoData {
+  id: string;
+  value: string;
+
+  constructor(id: string, value:any) {
```

Comment:
Why do you accept any value for the value field if the field itself is typesafe to string?